pkgs/by-name/eb/eb-garamond/package.nix

```diff
@@ -17,7 +17,8 @@ stdenvNoCC.mkDerivation rec {
   };
 
   nativeBuildInputs = [
-    (python3.withPackages (p: [ p.fontforge ]))
+    python3
+    python3.pkgs.fontforge
```

Comment:
Use `python3Packages.fontforge` for something something splicing reason. (ref: https://github.com/NixOS/nixpkgs/issues/211340)